Added timeline table

noatamir opened the pull request:
feel free to adjust the timeline - but I think this already looks clearer than several lists of meeting type ;)

noatamir committed:
changed email to low case b

noatamir committed:
MD timeline for all the scheduled meetings

krasch commented:
Sure does look much clearer!

Two things:
* Maybe put a note on top, that Tuesday/Thursday and the times are examples; the actual timeslots to be agreed-upon
* Link in handbook.md?

krasch commented:
Maybe add week_id as first column?

noatamir committed:
added week numbers and comment 

just to clarify we are open to changes. But this will give you a good idea of the time commitment and "flow".

noatamir commented:
I accidentally added the link on master 🤦‍♀️. So let's merge this if it's ok by you.

krasch committed:
Update timeline.md

krasch committed:
fix typos
